### ISSUE TITLE
can't switch the input method

### ISSUE THREAD

**Issue by lhish** [ibus/ibus#2614]
Which distribution and version?:
Ubuntu 22.04
Which desktop environment and version?:
gnome shell 42.9

Which application and version?:
All apps : (

IBus version?:
(Run ibus version)
1.5.26

Issue description:
can't switch the input method，my current solution to this is just leave the chinese input method and delete the english input method

Steps to reproduce:
dont know, it happens after i reboot my computer by the power buttom as it is dead. are there any log for me to see what the ibus-daemon do when i do the operation of switching the input method, i think this may help me find out the problem.

Can you reproduce your problem when you restart ibus-daemon? (yes / no):
yes

Do you see any errors when you run ibus-daemon with the verbose option?:
no

**Comment by lhish** [comment]
all the possible input method is showed properly but has no effect after i click one the input method that i wasn't using

**Comment by lhish** [comment]
and all the input method itself worked properly, normally , the default input method is english and it worked properly, and when i excute ibus-setup in terminal, the default input method in terminal changed into chinese which also worked properly in terminal. but after i use other app the default input method changed into english automatically

**Comment by fujiwarat** [comment]
> dont know, it happens after i reboot my computer by the power buttom as it is dead. are there any log for me to see what the ibus-daemon do when i do the operation of switching the input method

Can you explain how to switch your input method engines?

**Comment by lhish** [comment]
by clicking the ibus icon and the possible input methods will show, then by clicking these then the input method switch, such as switching from english to chinese. i also tried the shortcut keys, but nothing happened

**Comment by lhish** [comment]
> > dont know, it happens after i reboot my computer by the power buttom as it is dead. are there any log for me to see what the ibus-daemon do when i do the operation of switching the input method
> 
> Can you explain how to switch your input method engines?
> 

by clicking the ibus icon and the possible input methods will show, then by clicking these then the input method switch, such as switching from english to chinese. i also tried the shortcut keys, but nothing happened

**Comment by fujiwarat** [comment]
> by clicking the ibus icon and the possible input methods will show, then by clicking these then the input method switch, such as switching from english to chinese. i also tried the shortcut keys, but nothing happened

Your issue can be reproduce with both the mouse operation and shorcut key one?
I guessed your issue was cause by shortcut keys only.

**Comment by lhish** [comment]
> > by clicking the ibus icon and the possible input methods will show, then by clicking these then the input method switch, such as switching from english to chinese. i also tried the shortcut keys, but nothing happened
> 
> Your issue can be reproduce with both the mouse operation and shorcut key one? I guessed your issue was cause by shortcut keys only.

I may have realized the real cause of the problem after reading your reply, but I'm not completely sure. When I tried to check the settings, I used the "sudo ibus-setup" command, and surprisingly it showed that the ibus daemon wasn't running. However, before using the command, I'm very sure it was running, as I checked all the processes after rebooting. After using the command and clicking yes to run the daemon, the process list didn't change.

There are two more things to pay attention to:
1. The settings shown are different when I run "ibus-setup" and "sudo ibus-setup". Only the sudo version told me the ibus-daemon wasn't running.
2. I usually use the system input method tray icon to manage input methods. 
![image](https://github.com/ibus/ibus/assets/59965910/732a2719-eeba-46e7-970a-b022a92389da)
But after running "sudo ibus-setup" and clicking yes, a new tray icon appeared (but the original one didn't disappear). 
![截图 2024-03-02 07-08-15](https://github.com/ibus/ibus/assets/59965910/7556803d-dcb8-4720-838e-cfca7bc65c95)
I'm afraid I can't show the image of it after clicking the icon, as PrtSc had no effect then. In this new icon I had no problem switching between input methods by clicking on the options, but it had no effect on the input method I was actually using, which was still controlled by the original system tray icon i think.
